### PR TITLE
Adds hamburger menu on mobile to match design

### PR DESCRIPTION
**Pull request by polygonart6777**
[Fix for Issue 64](https://github.com/pyladies/global-conference/issues/63)

Adds hamburger menu on mobile to match the design given in [Figma design](https://www.figma.com/file/ow9vzSa50YsUDaI65RJB81/PyLadies-Conference?type=design&node-id=141-5845&mode=design&t=ZcEpcd8HBzVuVDSk-0)

Before:
Screenshot from 2023-10-21 19-59-28

After:
Screenshot from 2023-10-21 23-26-28

Desktop stays the same

### CODE REVIEW

**Comment by polygonart6777** [comment]
One issue I'm having trouble with:
When navigating from one section on the about page to another section via the menu, the menu should close and currently it stays open.  Example, navigate to the about page by clicking on Code of Conduct and then click on The Event link, notice that the menu stays open.  

Anyone have any ideas on how to fix this?

**Comment by Mariatta** [comment]
Thanks for working on this. Locally I'm not seeing a hamburger icon. Is there an image that needs to be committed?

File: generate_site/templates/header.html [hunk 1]
```diff
@@ -19,19 +19,39 @@
...
     <!--
     <div class="text-end">
         <a class="btn btn-outline-secondary btn-pink text-white" href="">Register</a>
     </div>
     -->
   </ul>
+
+  <nav class="navbar d-flex d-sm-none justify-content-between px-4">
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarToggler" aria-controls="navbarToggler" aria-expanded="false" aria-label="Toggle navigation">
+      <img class="hamburger-menu" src="img/bar-chart-2.png"/>
```
Comment (Mariatta) [inline generate_site/templates/header.html:40]: the img/bar-chart-2.png is missing.

**Comment by cmaureir** [comment]
With a width of 600px I don't see the menu
Screenshot 2023-11-03 at 10-33-09 PyLadiesCon 2023

File: generate_site/templates/header.html (hunk 1 above)
Comment (cmaureir) [reply generate_site/templates/header.html:40]: just push the change with the icon

**Comment by cmaureir** [comment]
Will merge in the meantime, thanks @polygonart6777 :tada:

**Comment by Mariatta** [comment]
Thank you for the work, @polygonart6777 and @cmaureir . I just checked the website on my phone, it the hamburger menu works. Now we can see all the different menu items.

**Pull request merged**